tools/dvrescue.xsd

```diff
@@ -44,36 +50,180 @@
-    <xsd:attribute name="size" type="xsd:string"/>
-    <xsd:attribute name="video_rate" type="xsd:string"/>
-    <xsd:attribute name="chroma_subsampling" type="xsd:string"/>
-    <xsd:attribute name="scanType" type="xsd:string"/>
-    <xsd:attribute name="aspect_ratio" type="xsd:string"/>
-    <xsd:attribute name="audio_rate" type="xsd:integer"/>
-    <xsd:attribute name="channels" type="xsd:integer"/>
+    <xsd:attribute name="scan_type" type="xsd:string">
+      <xsd:annotation>
+        <xsd:documentation xml:lang="en">
+          String representing the interlacement pattern of the frame.
+        </xsd:documentation>
+      </xsd:annotation>
+    </xsd:attribute>
+    <xsd:attribute name="count" type="xsd:integer">
+      <xsd:annotation>
+        <xsd:documentation xml:lang="en">
+          Frame number within the current frames sequence.
+        </xsd:documentation>
+      </xsd:annotation>
+    </xsd:attribute>
+    <xsd:attribute name="pts" type="xsd:string">
+      <xsd:annotation>
+        <xsd:documentation xml:lang="en">
+          Presentation timestamp for start of sequence.
+        </xsd:documentation>
+      </xsd:annotation>
+    </xsd:attribute>
+    <xsd:attribute name="end_pts" type="xsd:string">
+      <xsd:annotation>
+        <xsd:documentation xml:lang="en">
+          Presentation timestamp for end of sequence.
+        </xsd:documentation>
+      </xsd:annotation>
+    </xsd:attribute>
+    <xsd:attribute name="size" type="xsd:string">
+      <xsd:annotation>
+        <xsd:documentation xml:lang="en">
+          Dimensions of frame represented as width x height in pixels.
+        </xsd:documentation>
+      </xsd:annotation>
+    </xsd:attribute>
+    <xsd:attribute name="video_rate" type="xsd:string">
+      <xsd:annotation>
+        <xsd:documentation xml:lang="en">
+          Frame rate in fractional notation.
+        </xsd:documentation>
+      </xsd:annotation>
+    </xsd:attribute>
+    <xsd:attribute name="chroma_subsampling" type="xsd:string">
+      <xsd:annotation>
+        <xsd:documentation xml:lang="en">
+          Chroma subsampling value represented in three part ratio notation.
+        </xsd:documentation>
+      </xsd:annotation>
+    </xsd:attribute>
+    <xsd:attribute name="aspect_ratio" type="xsd:string">
+      <xsd:annotation>
+        <xsd:documentation xml:lang="en">
+          Frame aspect ratio represented in fractional notation.
+        </xsd:documentation>
+      </xsd:annotation>
+    </xsd:attribute>
+    <xsd:attribute name="audio_rate" type="xsd:integer">
+      <xsd:annotation>
+        <xsd:documentation xml:lang="en">
+          Frame audio rate represented in full integer notation.
+        </xsd:documentation>
+      </xsd:annotation>
+    </xsd:attribute>
+    <xsd:attribute name="channels" type="xsd:integer">
+      <xsd:annotation>
+        <xsd:documentation xml:lang="en">
+          Number of channels. Options are 1 (25  Mb/s  structure) or 2 (50  Mb/s  structure).
```

Comment:
options are wrong, can be 1 or 2 for DV25, between 1 and 4 for DV50, between 1 and 8 for DV100. 0 is possible in stream but in that case this attribute is not shown (should we show 0?).